packaging/bin/pki-enroll-openssh-cert.sh

```diff
@@ -32,6 +32,7 @@ sign() {
 		}
 		trap cleanup 0
 		cat "${PKIDIR}/private/ca.pem" > "${TMPCA}"
+		umask 0022
```

Comment:
Shouldn't it be before the openssl call above?

Reply:
it's just the public key, but yes, it would be "nicer"

Reply:
There is no reason to have public key files writable by everybody.  And changing a public key can be used to assist with redirection to false hosts or accounts.